stubs/olefile/olefile/olefile.pyi

```diff
@@ -122,7 +122,7 @@ DEFECT_INCORRECT: int
 DEFECT_FATAL: int
 MINIMAL_OLEFILE_SIZE: int
 
-def isOleFile(filename: IO[bytes] | bytes | str | None = None, data: bytes | None = None) -> bool: ...
+def isOleFile(filename: IO[bytes] | AnyStr | None = None, data: bytes | None = None) -> bool: ...
```

Comment:
In this case, `AnyStr` doesn't do anything. It's a type variable, and is appears only once in this definition, so using `bytes | str` was correct here.

Reply:
Ok, I've pushed a commit for this.